docs/sources/http_api/access_control.md

```diff
@@ -955,124 +955,39 @@ Content-Type: application/json; charset=UTF-8
-```
-
-#### Example response
-
-```http
-HTTP/1.1 200 OK
-Content-Type: application/json; charset=UTF-8
-
-{
-    "Admin": [
-        {
-            "version": 1,
-            "uid": "qQui_LCMk",
-            "name": "fixed:users:writer",
-            "name": "User writer",
-            "description": "Read and update all attributes and settings for all users in Grafana: update user information, read user information, create or enable or disable a user, make a user a Grafana administrator, sign out a user, update a user’s authentication token, or update quotas for all users",
-            "global": true,
-            "updated": "2021-05-13T16:24:26+02:00",
-            "created": "2021-05-13T16:24:26+02:00"
-        },
-        {
-            "version": 1,
-            "uid": "PeXmlYjMk",
-            "name": "fixed:users:reader",
-            "displayName": "User reader",
-            "description": "Allows every read action for user organizations and in addition allows to administer user organizations",
-            "global": true,
-            "updated": "2021-05-13T16:24:26+02:00",
-            "created": "2021-05-13T16:24:26+02:00"
-        }
-    ],
-    "Grafana Admin": [
-        {
-            "version": 1,
-            "uid": "qQui_LCMk",
-            "name": "fixed:users:writer",
-            "displayName": "User writer",
-            "description": "Read and update all attributes and settings for all users in Grafana: update user information, read user information, create or enable or disable a user, make a user a Grafana administrator, sign out a user, update a user’s authentication token, or update quotas for all users",
-            "global": true,
-            "updated": "2021-05-13T16:24:26+02:00",
-            "created": "2021-05-13T16:24:26+02:00"
-        }
-    ]
-}
-```
-
-#### Status codes
-
-| Code | Description                                                          |
-| ---- | -------------------------------------------------------------------- |
-| 200  | Built-in role assignments are returned.                              |
-| 403  | Access denied                                                        |
-| 500  | Unexpected error. Refer to body and/or server logs for more details. |
-
-### Create a built-in role assignment
-
-`POST /api/access-control/builtin-roles`
-
-Creates a new built-in role assignment.
-
-#### Required permissions
+<!-- TODO rephrase double negation might be hard to understand -->
 
-`permissions:type:delegate` scope ensures that users can only create built-in role assignments with the roles which have same, or a subset of permissions which the user has.
-For example, if a user does not have required permissions for creating users, they won't be able to create a built-in role assignment which will allow to do that. This is done to prevent escalation of privileges.
+`permissions:type:escalate` scope does not ensure that users don't acquire more permissions with the reset.
```

Comment:
@chri2547 isn't the double negation here to hard to understand?

Reply:
yes, that is confusing. what are we trying to say about this scope?

Reply:
While
> "`permissions:type:delegate` scope ensures that users can only create built-in role assignments with the roles which have same, or a subset of permissions which the user has."
> `permissions:type:escalate` doesn't. Users could acquire more permissions than they already have.

Reply:
Right now it means that they can reset roles, even if that means users might acquire new permissions.

Reply:
I'll just remove the comment, it's not that relevant.

Reply:
Can we say:

`permissions:type:escalate` scope enables users to create basic role assignments that include permissions that exceed the permissions of the current role.

Reply:
This is the best I could do :sweat_smile:: 
```
`permissions:type:escalate` scope enables users to reset basic roles permissions.
This could result in basic roles having permissions exceedind those of callers.

```